### ISSUE TITLE
getAllData() doesn't return networkStats for all interfaces.

### ISSUE THREAD

**Issue by credomane** [sebhildebrandt/systeminformation#417]
With the change to networkStats to support the wildcard "*" to get network stats for all interfaces (#181) it would be nice if getAllData() would make use of this wildcard as currently it returns the default interface only.

**Comment by sebhildebrandt** [comment]
@credomane  As `networkStats()` also defaults to the default interface, I implemented the same behaviour for `getAllData()`. So if you want getAllData to return data for all interfaces, you can easily manage this by calling it like 

```
si.getAllData('apache, ...', '*').then(...
```

This is I guess more consistent. Makes sense??

But anyway, if you really want to get network stats periodically, you should call this in a loop as explained in https://systeminformation.io/statsfunctions.html

**Comment by credomane** [comment]
I'm using the stat functions already, BTW.  SystemInformation is awesome! Wish I discovered it sooner.

Best I can remember from last week is I was trying to see what all this package could see/discover using getAllData and specifically noticed it wasn't returning all network data. Looked to docs, couldn't find getAllData() so I went to the source and saw this line https://github.com/sebhildebrandt/systeminformation/blob/3175da35b5b8a2372f5c1130d8f0a5ed86cdc90d/lib/index.js#L138 and thought maybe this should use `'*'` now instead of `network.getDefaultNetworkInterface()` for the "fallback" value since that is possible now.

Had I just looked how iface was defined or looked at the docs closer I would have discovered the arguments for getAllData(). Take it from me don't read source distracted and/or tired. Sorry for wasting your time on this report.
